become-a-karakuni.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: article
+title: You@Karakun
+order: '19'
+headerText: 
+permalink: /become-a-karakuni/
+header:
+  image: team
+  text: Work <span class="my-karakun">happier!</span>
+nav:
+  bottom: true
+---
+
+## Love where you work
+
+At Karakun, we’re looking for talented people to do work that they love at all levels of the company. 
+TODO: allgeminer netter Text über Karakun
+
+## Stay up to date
+
+The following image shows just some of the technologies and open source projects that we at Karakun use in
+our daily business. Since we love open source we already contributed to several of that technologies.
+
+![technologies]({{ "/assets/images/pages/become-a-karakuni/technologies.png" | absolute_url }})
+
+TODO: Schulung, Vorträge, Mit Autoren & JCs zusammen arbeiten. Contribution to open source
+
+## Find the right location
+
+TODO: Basel oder Dortmund
```

Comment:
Or Stuttgart :P

Reply:
No :P Currently only for the 2 headquarters

Reply:
We only have one headquater - Basel ;-)